pmd-core/src/main/java/net/sourceforge/pmd/Report.java

```diff
@@ -512,7 +522,7 @@ public long getElapsedTimeInMillis() {
         return end - start;
     }
 
-    public List<SynchronizedReportListener> getSynchronizedListeners() {
+    public List<ThreadSafeReportListener> getSynchronizedListeners() {
```

Comment:
I was not sure, since I'm already breaking the API, if I should rename this method right now or not. What do you think @adangel ?